lib/components/viewers/live-stop-times.tsx

```diff
@@ -146,24 +153,33 @@ class LiveStopTimes extends Component<Props, State> {
 
     return (
       <>
-        <div>
+        <ul className="route-row-container">
           {Object.values(stopTimesByPattern)
             .sort(patternComparator)
+            .filter(({ pattern, route }) =>
+              routeIsValid(route, getRouteIdForPattern(pattern))
+            )
             .map(({ id, pattern, route, times }) => {
               // Only add pattern if route info is returned by OTP.
-              return routeIsValid(route, getRouteIdForPattern(pattern)) ? (
+              return (
                 <PatternRow
                   homeTimezone={homeTimezone}
                   key={id}
                   pattern={pattern}
-                  route={route}
+                  route={{
+                    ...route,
+                    operator: transitOperators.find(
```

Comment:
It could be, but we'd then have to assemble it back together within the file for `next-arrival-for-pattern`. We can't ship operator as its own prop there since the related stops panel deals with a list of routes. Matching those arrays up would get messy.